src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -734,7 +734,7 @@ private void SetupSolutionBinding(bool isConnected, IList<SonarQubeIssue> issues
                     })
                 .Verifiable();
 
-            mockSqService.Setup(x => x.GetSuppressedIssuesAsync(It.IsAny<string>(), It.IsAny<CancellationToken>()))
+            mockSqService.Setup(x => x.GetSuppressedIssuesAsync(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<CancellationToken>()))
```

Comment:
I haven't added any new tests, just fixed up the existing ones so they don't fail.